src/core/reducers/api.js

```diff
@@ -20,16 +20,21 @@ import type {
 import type { Exact } from 'core/types/util';
 
 type UserAgentInfoType = {|
-  browser: string,
-  os: string,
+  browser: {
+    name?: string,
+    version?: string,
```

Comment:
Thanks, this is really helpful. Huh, did `UAParser` change? Or maybe this was always typed wrong. Could you add in the other property? I see it like this in the console:

```js
browser: {
  name?: string,
  version?: string,
  major?: string,
}
```